build/dependencies.props

```diff
@@ -2,13 +2,13 @@
   <PropertyGroup>
     <DotNetCliUtilsVersion>1.0.1</DotNetCliUtilsVersion>
     <DotNetProjectModelVersion>1.0.0-rc3-1-003177</DotNetProjectModelVersion>
-    <HtmlAgilityPackVersion>1.5.0-beta8</HtmlAgilityPackVersion>
+    <HtmlAgilityPackVersion>1.5.1</HtmlAgilityPackVersion>
     <!-- This one is OK for console tools that bundle their own version of JSON.NET -->
     <JsonNetVersion>10.0.1</JsonNetVersion>
-    <MicrosoftDotNetPlatformAbstractionsVersion>1.1.0</MicrosoftDotNetPlatformAbstractionsVersion>
-    <MonoCecilVersion>0.10.0-beta1-v2</MonoCecilVersion>
+    <MicrosoftDotNetPlatformAbstractionsVersion>2.0.0</MicrosoftDotNetPlatformAbstractionsVersion>
+    <MonoCecilVersion>0.10.0-beta6</MonoCecilVersion>
```

Comment:
It's equivalent in the BCL is System.Reflection.Metadata. I haven't looked into it, but I've been told the API isn't as friendly as Cecil.